gravityflow/gravityflowformconnector
# New Step Type - Update Field Values

The existing Update an Entry step modifies values INTO the chosen entry whereas this new step type will update field values FROM a chosen entry.

**To Test:**
Execute the Codeception test 0005 which:
- Verifies both local and remote for Update Fields step type
- Verifies both local and remote for Update an Entry step type (negative test in context of PR)

Execute the Codeception test 0006 which:
- Incorporates support for GForm PR #190 for conditional logic entry ID lookups.
- Verifies Update Fields step type pulls expected data for Local By ID, Local By Entry Filter, Remote By ID, Remote By Entry Filter
- Verifies Update Fields step type pulls expected data involving Sort Criteria for Local + Remote By Entry Filter.
